packages/expo-dev-launcher/bundle/native-modules/DevLauncherInternal.ts

```diff
@@ -44,3 +51,117 @@ export async function copyToClipboardAsync(content: string): Promise<null> {
 export const clientUrlScheme = DevLauncher.clientUrlScheme;
 export const installationID = DevLauncher.installationID;
 export const isDevice = !!DevLauncher.isDevice;
+
```

Comment:
do you think there is value in keeping the auth session / web browser stuff as a separate native module, even if it lives inside the dev launcher package?

Reply:
I think that makes a lot of sense